po/LINGUAS

```diff
@@ -37,3 +37,4 @@ zh_CN
 zh_TW
 eu
 he
+zh_Hans
```

Comment:
Hmm, shouldn't we treat zh_CN the same as zh_Hans? The former is already fully translated...

Reply:
Yeah, and this translation is empty..

Reply:
Now it does have proper translation, but looks like a complete duplicate of zh_CN

cc @keszybz

Reply:
Indeed, the translations are identical. I was trying to log into the Weblate interface to sort this out, but there appears to be some FAS outage, so it'll have to wait until that's resolved.

Reply:
> I'm, however, not sure about other implications this could have, mainly concerning gettext, but I guess specifying both the legacy and new identifiers in the LINGUAS file should work fine?

There're no `zh_Hans` or `zh_Hant` locales AFAIK. The locales have always been `zh_CN` and `zh_TW`. That said, I've never done translation work before and have poor knowledge about how gettext works :(

Reply:
@mrc0mmand, Hello. Like you said in the previous message, the case is a little complicated. Some projects prefer the new locale names, zh_Hans and zh_Hant, for example Fedora Websites and Fedora Docs, the others like Fedora installer Anaconda, sticks to the old ones zh_CN and zh_TW. 

There was a similar discuss on the Fedora L10n mailing list in 2020, the result was Fedora Websites continued to use the new language codes. I didn't find any L10n impacts or bugs till now. I hope you will get some useful information here https://lists.fedoraproject.org/archives/list/trans@lists.fedoraproject.org/thread/DPAF4M2YOPAMPLVCELI665G4LUP2J44N/ 

Translators will have to switch between zh_Hans and  zh_CN or zh_Hant and zh_TW if they contribute to multiple projects on Weblate. This is not a big issue to experienced translators, but may make new translators get confused. So it would be better to add some note after you switch to the new codes.

For end users, I think a simple mapping will handle it, as long as they will not notice the difference.(Maybe I am too optimistic)

The hardest part is on the developing side. I am a layman, but I am sure systemd development is quite different from website and docs. I don't know if the change will cause any issue. What about ask the maintainers of a project which has similar development process to systemd and uses the new language codes?